src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java

```diff
@@ -93,36 +161,99 @@ public void testGVCFModeIsConsistentWithPastResults() throws Exception {
     }
 
     /*
+ * Test that in GVCF mode we're consistent with past GATK4 results using AS_ annotations
+ */
+    @Test
+    public void testGVCFModeIsConsistentWithPastResults_AlleleSpecificAnnotations() throws Exception {
+        Utils.resetRandomGenerator();
+
+        final File output = createTempFile("testGVCFModeIsConsistentWithPastResults_AlleleSpecificAnnotations", ".g.vcf");
+        final File expected = new File(TEST_FILES_DIR + "expected.testGVCFMode.gatk4.alleleSpecific.g.vcf");
+
+        final String[] args = new String[] {
+                "-I", NA12878_20_21_WGS_bam,
+                "-R", b37_reference_20_21,
+                "-L", "20:10000000-10100000",
+                "-O", output.getAbsolutePath(),
+                "-G", "StandardAnnotation",
+                "-G", "AS_StandardAnnotation",
+                "-ERC", "GVCF",
+                "-pairHMM", "AVX_LOGLESS_CACHING"
+        };
+
+        runCommandLine(args);
+
+        // Test for an exact match against past results
+        IntegrationTestSpec.assertEqualTextFiles(output, expected);
+    }
+
+    /*
      * Test that in GVCF mode we're >= 99% concordant with GATK3 results
      */
     @Test
-    public void testGVCFModeIsConcordantWithGATK3Results() throws Exception {
+    public void testGVCFModeIsConcordantWithGATK3_5Results() throws Exception {
         Utils.resetRandomGenerator();
 
         final File output = createTempFile("testGVCFModeIsConcordantWithGATK3Results", ".g.vcf");
-        final File gatk3Output = new File(TEST_FILES_DIR, "expected.testGVCFMode.gatk3.4.g.vcf");
```

Comment:
You left the previous expected output `expected.testGVCFMode.gatk3.4.g.vcf` checked into the test directory -- should we delete it, or is it worth keeping around for a while for troubleshooting purposes?